scripts/shell/init_seednode.sh

```
@@ -0,0 +1,56 @@
+# Initialize first bootnode
+echo "Initializing bootnode-1..."
+sudo rm -rf /home/ubuntu/.treasurenetd
+treasurenetd init bootnode-1 --chain-id "treasurenet_5570-1"
+
+# Set up data directory for bootnode-1
+sudo mkdir -p /data/bootnode-1/.treasurenetd
+sudo rm -rf /data/bootnode-1/.treasurenetd
+sudo mv /home/ubuntu/.treasurenetd /data/bootnode-1/.treasurenetd
+
+# Copy genesis configuration from validator
+cd /data/genesis-validator-1/.treasurenetd/config/
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)